c++/src/kj/refcount.h

```diff
@@ -89,7 +90,7 @@ class Refcounted: private Disposer {
   static Own<T> addRefInternal(T* object);
 
   template <typename T>
-  static Rc<T> addRcRefInternal(T* object);
+  static Rc<T> addPtrRefInternal(T* object);
```

Comment:
Why the rename?

Reply:
I needed the same name to make `EnableAddRefToThis` work with both `Refcounted` and `AtomicRefcounted`